wiki/Medals/ru.md

```diff
@@ -1,58 +1,67 @@
 
-Медали группы **Beatmap Packs** могут быть получены за **игру на каждой карте до конца хотя бы на одной из сложностей** из [тематических сборок](https://osu.ppy.sh/beatmaps/packs?type=theme). Чтобы получить статус **`Пройдено`** в [списке карт сборки](https://osu.ppy.sh/beatmaps/packs?type=theme) и значок, вы обязаны играть в режиме [osu!](/wiki/Game_mode/osu!).
+Медали группы **Beatmap Packs** могут быть получены за **прохождение до конца хотя бы одной из сложностей на каждой карте** из [Тематических Сборок](https://osu.ppy.sh/beatmaps/packs?type=theme) или [Сборок Исполнителей/Альбомов](https://osu.ppy.sh/beatmaps/packs?type=artist). 
 
-### Требования
+Все карты должны быть пройдены в режиме игры [osu!](/wiki/Game_mode/osu!) для получения значка, если не указано обратное. Использование [упрощающих модов](/wiki/Game_modifier#difficulty-reduction) или [усложняющих модов](/wiki/Game_modifier#difficulty-increase), как и использование [Spun Out](/wiki/Game_modifier/Spun_Out) разрешено.
 
-- Игрок обязан закончить каждую карту сборки, загруженной из [тематических сборок](https://osu.ppy.sh/beatmaps/packs?type=theme) один раз на любой сложности.
-- Игра обязана быть в режиме osu!.
-- Распакуйте тематическую сборку из `.zip`-файла и поместите её содержимое в папку `Songs`.
-  - Затем откройте клиент osu! и позвольте клиенту проанализировать файлы перед игрой на них.
-  - Кроме того, игрок может загружать песни вручную одну за другой, и это все равно будет засчитываться.
-- Какие-либо вариации (вроде другого создателя) песни не засчитываются.
-  Это **обязана** быть именно та указанная песня, тот указанный создатель и та указанная карта.
-- Игрок, о котором идёт речь, должен произвести вход в клиенте osu! (не на сайте).
+Однако стоит отметить что содержимое всех Video Game, Anime, Internet! и Rhythm Game Паков сильно изменилось с [их первого появления в 2008](https://osu.ppy.sh/community/forums/topics/1853). Для списка этих паков в их оригинальном состоянии см. [Legacy Beatmap Packs](/wiki/Medals/Legacy_beatmap_packs).
 
-### Примечания
+## Skill
 
-- Разрешено использовать моды, за исключением тех, что делают прохождение безрейтинговым (например, [Auto Pilot](/wiki/Game_modifier/Autopilot)).
-- Карта может быть без проблем удалена, как только её статус в [списке карт сборки](https://osu.ppy.sh/beatmaps/packs?type=theme) будет отмечен как **`Пройдено`**.
+Медали группы **Skill** присуждаются за достижения, основанные на различных навыках, будь это получение определённого [комбо](/wiki/Beatmapping/Combo), достижение определённого ранга в [глобальной таблице лидеров по очкам производительности](https://osu.ppy.sh/rankings/osu/performance), прохождение или [полное комбо](/wiki/Gameplay/Full_combo) на любой [Квалифицированной](/wiki/Beatmap/Category#qualified), [Рейтинговой](/wiki/Beatmap/Category#ranked) или [Одобренной](/wiki/Beatmap/Category#approved) карте в определённом диапазоне [звёздного рейтинга](/wiki/Beatmapping/Star_rating).
```

Comment:
```suggestion
Медали группы **Skill** присуждаются за достижения, основанные на различных навыках, будь это получение определённого [combo](/wiki/Beatmapping/Combo), достижение определённого ранга в [глобальной таблице лидеров по очкам производительности](https://osu.ppy.sh/rankings/osu/performance), прохождение или [full combo](/wiki/Gameplay/Full_combo) на любой [Qualified](/wiki/Beatmap/Category#qualified), [Ranked](/wiki/Beatmap/Category#ranked) или [Approved](/wiki/Beatmap/Category#approved) карте в определённом диапазоне [star rating](/wiki/Beatmapping/Star_rating).
```
-> refering to (https://osu.ppy.sh/wiki/ru/Beatmap/Category), categories don't seems to be translated.
    -> also technical stuff like combo / full combo / star rating don't have to be translated (i guess ? see french translation)